dotcom-rendering/src/types/configContext.ts

```diff
@@ -8,15 +9,17 @@ import type { RenderingTarget } from './renderingTarget';
  * @see /dotcom-rendering/docs/architecture/proposed-adrs/react-context-api.md
  */
 export type Config =
-	| {
+	| Readonly<{
 			renderingTarget: Extract<RenderingTarget, 'Web'>;
 			darkModeAvailable: boolean;
 			updateLogoAdPartnerSwitch: boolean;
 			assetOrigin: AssetOrigin;
-	  }
-	| {
+			editionId: EditionId;
```

Comment:
We have a rule in the sand for our config use that [Context should be considered **global, static, and immutable** ](https://github.com/guardian/dotcom-rendering/blob/85e286cc5b36697f49e9373b993b3ad3a598a641/dotcom-rendering/docs/architecture/028-react-context-api.md?plain=1#L15C5-L15C68) - Does this violate that?